### DISCUSSION TITLE
Specification for UR type 'crypto-psbt'

### DISCUSSION THREAD

**Discussion by craigraw** [BlockchainCommons/Gordian-Developer-Community#11] [General & Announcements]
Hi,

As discussed just opened a PR for representing PSBTs in UR format: https://github.com/BlockchainCommons/Research/pull/37

As the PR notes, I didn't feel there would be value in trying to copy data out of the PSBT into CBOR, and PSBTs are meant to be self-contained anyway. So I have used a simple CBOR byte string for maximal efficiency, given PSBT is a binary format. The value of the specification lies in knowing that you are dealing with a PSBT from the UR type.

Craig

**Comment by Fonta1n3** [comment]
Thanks Craig, I certainly agree with this approach and look forward to adding it into my projects.

**Reply by craigraw** [reply]
Thanks @Fonta1n3 - Chris is asking for another ACK on the PR before merging if you are happy to proceed.

**Comment by wolfmcnally** [comment]
LGTM
